sample-apps/java-sample-app/manual/src/main/java/software/amazon/adot/sampleapp/AwsXrayContextDataProvider.java

```diff
@@ -0,0 +1,32 @@
+package software.amazon.adot.sampleapp;
+
+        import io.opentelemetry.api.trace.Span;
+        import io.opentelemetry.api.trace.SpanContext;
+        import java.util.Collections;
+        import java.util.Map;
+        import org.apache.logging.log4j.core.util.ContextDataProvider;
```

Comment:
nit: do these imports have a tab in front or is the GitHub UI being weird?

Reply:
I will fix.